crates/polars/tests/it/io/ipc_stream.rs

```diff
@@ -9,88 +9,113 @@ mod test {
 
     use crate::io::create_df;
 
-    #[test]
-    fn write_and_read_ipc_stream() {
-        // Vec<T> : Write + Read
-        // Cursor<Vec<_>>: Seek
+    fn create_ipc_stream(mut df: DataFrame) -> Cursor<Vec<u8>> {
         let mut buf: Cursor<Vec<u8>> = Cursor::new(Vec::new());
-        let mut df = create_df();
 
         IpcStreamWriter::new(&mut buf)
             .finish(&mut df)
-            .expect("ipc writer");
+            .expect("failed to write ICP stream");
 
         buf.set_position(0);
 
-        let df_read = IpcStreamReader::new(buf).finish().unwrap();
-        assert!(df.equals(&df_read));
+        buf
+    }
+
+    macro_rules! assert_df_eq {
+        ($a:expr, $b:expr) => {
+            let a: &polars_core::frame::DataFrame = &$a;
+            let b: &polars_core::frame::DataFrame = &$b;
+            assert!(a.equals(b), "expected {:?}\nto equal {:?}", a, b);
+        };
+    }
```

Comment:
Does this rust macro to assert data frames equality already exist somewhere?

Should this be here?

Reply:
Nope, but if we do define it it shouldn't be here. We have a testing module in `polars-core`.

Reply:
Moved the macro declaration to `polars_core::testing`.